WordPress/Classes/ViewRelated/Post/PostEditorAnalyticsSession.swift

```diff
@@ -40,6 +40,11 @@ struct PostEditorAnalyticsSession {
             let blocksJSON = String(data: data, encoding: .utf8)
             properties[Property.unsupportedBlocks] = blocksJSON
         }
+
+        if let canViewEditorOnboarding = capabilities["canViewEditorOnboarding"] {
```

Comment:
Is there a better approach than using a string literal to access this key?

Reply:
I don't think there's a better way. Converting the capabilities dictionary back to `[Capabilities: Bool]` wouldn't help much.